kubeflow/core/tests/jupyterhub_test.jsonnet

```diff
@@ -130,6 +131,12 @@ std.assertEqual(jupyterhub.parts(params.namespace).jupyterHub(params.jupyterHubI
                      "-f",
                      "/etc/config/jupyterhub_config.py"
                   ],
+                  "env": [
+                     {
+                        "name": "NOTEBOOK_PVC_MOUNT",
+                        "value": "/home/jovyan/work"
```

Comment:
shouldn't the value be params.jupyterNotebookPVCMount?

Reply:
It's test validation and this path is default param.